scaffolder-templates/python-backend-template/skeleton/README.md

```diff
@@ -2,4 +2,4 @@
 
 ${{ values.description }}
 
-More information on the chosen CI method can be found [here](https://github.com/${{ values.githubOrg }}/${{ values.repoName }}/blob/main/CI.md).
+More information on the chosen CI method can be found [here](https://${{ values.sourceControl }}/${{ values.orgName }}/${{ values.repoName }}/blob/main/CI.md).
```

Comment:
Is this supposed to be just an example? Because `CI.md` file doesn't exist in the skeleton.